frontend/app/src/people/main/Body.tsx

```diff
@@ -85,6 +100,29 @@ function BodyComponent() {
     }
   }, [main, ui.meInfo]);
 
+  interface CodingLanguage {
+    [language: string]: boolean;
+  }
+
+  const filterByCodingLanguage = (users: PersonType[], codingLanguages: CodingLanguage) => {
+    const requiredLanguages = Object.keys(codingLanguages).filter(
+      (key: string) => codingLanguages[key]
+    );
+
+    return users.filter((user: PersonType) => {
+      const userCodingLanguages = (user.extras.coding_languages ?? []).map(
+        (t: { [key: string]: string }) => t.value
+      );
+      return requiredLanguages?.every((requiredLanguage: string) =>
+        userCodingLanguages.includes(requiredLanguage)
+      );
+    });
+  };
```

Comment:
we should put this in a util and write a test for it

Reply:
Okay @kevkevinpal